docs/composedb/guides/data-modeling/composites.mdx

```diff
@@ -65,16 +65,16 @@ const ceramic = new CeramicClient('http://localhost:7007')
 ceramic.did = did
 
 // Replace by the path to the source schema file
-const composite = await createComposite(ceramic, './source-schema.graphql')
+const composite = await createComposite(ceramic, './my-schema.graphql')
 
 // Replace by the path to the encoded composite file
-await writeEncodedComposite(composite, './my-composite.json')
+await writeEncodedComposite(composite, './__generated__/definition.json')
```

Comment:
calling this json file `definition` might cause confusion with the definition file resulting after running `writeEncodedCompositeRuntime`, I think it would be a good idea to leave it as `my-composite.json`